server/Mistakes.Journal.Api/Logic/Mistakes/Models/AdditionalQuestions.cs

```diff
@@ -0,0 +1,11 @@
+namespace Mistakes.Journal.Api.Logic.Mistakes.Models
+{
+    public class AdditionalQuestions
+    {
+        public string Consequences { get; set; }
+        public string WhatCanIDoBetter { get; set; }
+        public string WhatDidILearn { get; set; }
+        public string CanIFixIt { get; set; }
+        public string OnlyResponsible { get; set; }
+    }
```

Comment:
I think this class should be nested inside Mistake class. Otherwise, I would name it MistakeAdditionalQuestions.